packages/connect-web-test/src/interceptor.spec.ts

```diff
@@ -1,238 +0,0 @@
-// Copyright 2021-2024 The Connect Authors
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-import type {
-  CallOptions,
-  Interceptor,
-  StreamResponse,
-} from "@connectrpc/connect";
-import {
-  createCallbackClient,
-  createPromiseClient,
-  encodeBinaryHeader,
-} from "@connectrpc/connect";
-import { TestService } from "./gen/connectrpc/conformance/v1/test_connect.js";
-import { StreamingOutputCallRequest } from "./gen/connectrpc/conformance/v1/messages_pb.js";
-import { describeTransports } from "./helpers/conformanceserver.js";
-
-function makeLoggingInterceptor(name: string, log: string[]): Interceptor {
-  /**
-   * Return all keys of a Headers object, without needing
-   * DOM.iterable for Headers.keys().
-   */
-  function listHeaderKeys(header: Headers): string[] {
-    const fieldsToIgnore = [
-      "access-control-allow-origin",
-      "access-control-expose-headers",
-      "connect-accept-encoding",
-      "grpc-accept-encoding",
-      "grpc-message",
-      "grpc-status",
-      "content-type",
-      "date",
-      "transfer-encoding",
-      "vary",
-      "accept-encoding",
-      "content-encoding",
-      "content-length",
-      "connection",
-      "keep-alive",
-      "access-control-allow-headers",
-      "access-control-allow-methods",
-      "access-control-max-age",
-      "request-protocol",
-    ];
-    const keys: string[] = [];
-    header.forEach((_, key) => {
-      if (!fieldsToIgnore.includes(key)) {
-        keys.push(key);
-      }
-    });
-    return keys;
-  }
-
-  async function* logStream(res: StreamResponse) {
-    for await (const m of res.message) {
-      log.push(`${name} response stream received message`);
-      yield m;
-    }
-    yield* res.message;
-    log.push(
-      `${name} response stream done with trailers: ${listHeaderKeys(
-        res.trailer,
-      ).join(", ")}`,
-    );
-  }
-
-  return (next) => async (req) => {
-    log.push(`${name} sending request message`);
-    const res = await next(req);
-    const headerKeys = listHeaderKeys(res.header).join(", ");
-    log.push(`${name} response received with headers: ${headerKeys}`);
-    if (res.stream) {
-      return {
-        ...res,
-        message: logStream(res),
-      };
-    } else {
-      const trailerKeys = listHeaderKeys(res.trailer).join(", ");
-      log.push(`${name} response done with trailers: ${trailerKeys}`);
-    }
-    return res;
-  };
-}
-
-describe("unary interceptors", () => {
```

Comment:
These tests are not _great_, but it seems important to keep some test coverage for interceptors in a realistic scenario.